openpype/plugins/publish/extract_review_slate.py

```diff
@@ -314,6 +308,41 @@ def process(self, instance):
                 slate_subprocess_cmd, shell=True, logger=self.log
             )
 
+            # Create slate with silent audio track
+            if input_audio:
+                # silent slate output path
+                slate_silent_path = slate_path.replace(".png", "Silent" + ext)
```

Comment:
Longer but more safe proof to reuse `slate_v_path`.
```suggestion
                basename, ext = os.path.splitext(slate_v_path)
                slate_silent_path = "{}_silent{}".format(basename, ext)
```

Or singleline option
```
slate_silent_path = "_silent".join(os.path.splitext(slate_v_path))
```

Reply:
will do